stage1/init/common/pod.go

```diff
@@ -467,6 +467,47 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		return errwrap.Wrap(fmt.Errorf("failed to write app %q reaper service", appName), err)
 	}
 
+	if err = writeShutdownService(p); err != nil {
+		return errwrap.Wrap(errors.New("failed to write shutdown service"), err)
+	}
```

Comment:
Shouldn't this be in PodToSystemd() instead of appToSystemd()? In a pod with several apps, the shutdown service should still be written only one time.